frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-row.tsx

```diff
@@ -153,7 +165,7 @@ export const DiskRow: React.FC<VMDiskRowProps> = ({
 }) => {
   return (
     <DiskSimpleRow
-      data={restData}
+      data={{ ...restData, namespace: getNamespace(vmLikeEntity) }}
```

Comment:
another option would be to pass it in `getStoragesData` function to have the data comply with the StorageBundle type